### PR TITLE
Integrate 7/14 Nightly RN Build

### PR DESCRIPTION
**Pull request by NickGerleman**
Commits: https://github.com/facebook/react-native/compare/4ad4426da...ff9c8cdac

No non-mechanical changes needed

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/8448)

### CODE REVIEW

**Review by Khalef1 — APPROVED** [review]

File: packages/@react-native/tester/js/types/RNTesterTypes.js [hunk 1]
```diff
@@ -10,31 +10,32 @@
 
 import * as React from 'react';
 
-export type RNTesterExampleModuleItem = $ReadOnly<{|
+export type RNTesterModuleExample = $ReadOnly<{|
   name?: string,
   title: string,
-  platform?: string,
+  platform?: 'ios' | 'android',
```
Comment (dannyvv) [inline packages/@react-native/tester/js/types/RNTesterTypes.js:16]: should this include MacOS and Windows?

**Review by dannyvv — APPROVED** [review]
:shipit:

**Comment by dannyvv** [comment]
Just double-checking that the title of 7/14 is correct and it is not 8/14...

**Comment by dannyvv** [comment]
> Just double-checking that the title of 7/14 is correct and it is not 8/14...

Looks like valid, the last one was 7/7, apologies for the noise

File: packages/@react-native/tester/js/types/RNTesterTypes.js (hunk 1 above)
Comment (NickGerleman) [reply packages/@react-native/tester/js/types/RNTesterTypes.js:16]: We have an issue somewhere for adding other platforms to the selector here... For now though we haven't added our own, so the change is okay to take in.

**Pull request merged**